src/models/EventLogging.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum EventLoggingEnum {
+  ENQUIRY_UPDATE_NOP_INITIATED = 'ENQUIRY_UPDATE_NOP_INITIATED',
+  ENQUIRY_UPDATE_NOP_INITIATED_FOR_RECORD_ID = 'ENQUIRY_UPDATE_NOP_INITIATED_FOR_RECORD_ID',
+  ENQUIRY_UPDATE_NOP_FAILED = 'ENQUIRY_UPDATE_NOP_FAILED',
+  ENQUIRY_UPDATE_NOP_SUCCESSFUL = 'ENQUIRY_UPDATE_NOP_SUCCESSFUL',
```

Comment:
is there a way to separate the success log into 2, one for test results and one for tech-records?